src/plugins/ReactIntegrationUsageDataPlugin.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
I'm unsure if I didn't mention this somewhere already, but Eslint is complaining about the license key:
```Definition for rule 'ckeditor5-rules/prevent-license-key-leak' was not found```